homeassistant/components/tuya/__init__.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/env python3
+"""Support for Tuya Smart devices."""
+
+import itertools
+import json
+import logging
+from typing import Any
+
+from tuya_iot import (
+    ProjectType,
+    TuyaDevice,
+    TuyaDeviceListener,
+    TuyaDeviceManager,
+    TuyaHomeManager,
+    TuyaOpenAPI,
+    TuyaOpenMQ,
+    tuya_logger,
+)
+import voluptuous as vol
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.core import HomeAssistant
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+
+from .aes_cbc import AES_ACCOUNT_KEY, KEY_KEY, XOR_KEY, AesCBC as Aes
+from .const import (
+    CONF_ACCESS_ID,
+    CONF_ACCESS_SECRET,
+    CONF_APP_TYPE,
+    CONF_COUNTRY_CODE,
+    CONF_ENDPOINT,
+    CONF_PASSWORD,
+    CONF_PROJECT_TYPE,
+    CONF_USERNAME,
+    DOMAIN,
+    TUYA_DEVICE_MANAGER,
+    TUYA_DISCOVERY_NEW,
+    TUYA_HA_DEVICES,
+    TUYA_HA_TUYA_MAP,
+    TUYA_HOME_MANAGER,
+    TUYA_MQTT_LISTENER,
+    TUYA_SETUP_PLATFORM,
+    TUYA_SUPPORT_HA_TYPE,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
The current Tuya integration doesn't have a YAML configuration. We shouldn't add one for the v2 and mark it as deprecated, just drop it.

Reply:
Please remove the `CONFIG_SCHEMA`.